staging/src/k8s.io/apiserver/pkg/server/openapi/openapi.go

```diff
@@ -107,6 +157,40 @@ func (o *openAPI) init(webServices []*restful.WebService) error {
 			return err
 		}
 	}
+	if err := o.makeAdditionalFormats(); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func (o *openAPI) makeAdditionalFormats() error {
```

Comment:
Good point. I like methods to be self explanatory so instead of adding comments to this, I restructure it to two `toProtoBinary` and `toGzip` method that is clear what they do.